### PR TITLE
Added speech noise experiments plus other tweaks

### PR DESCRIPTION
**Pull request by subutai**

### CODE REVIEW

File: projects/speech_commands/run_experiment.py [hunk 1]
```diff
@@ -40,9 +43,24 @@
   projectDir = path.dirname(suite.options.config)
   dataDir = path.join(path.abspath(projectDir), "data")
 
-  if not path.isdir(path.join(dataDir, "speech_commands")):
-    tar = tarfile.open(path.join(dataDir, "speech_commands.tar.gz"))
-    tar.extractall(path=dataDir)
-    tar.close()
+  if "DOMINO_WORKING_DIR" in os.environ:
```
Comment (lscheinkman) [inline projects/speech_commands/run_experiment.py]: Not for tomorrow, but should we use the 'domino' dataset instead of this code?

File: projects/speech_commands/experiments.cfg [hunk 2]
```diff
@@ -77,23 +77,51 @@ model_type = "linear"
...
+learning_rate = 0.04
+momentum = 0.9
+weight_sparsity = 1.0
+dropout = 0
+log_interval = 40
+test_noise_every_epoch = False
+batches_in_epoch = 214
+batch_size = 96
+model_type = "resnet9"
+seed = [51, 52]
```
Comment (lscheinkman) [inline projects/speech_commands/experiments.cfg]: This is not necessary. The seed is updated on the reset based on the repetition.
https://github.com/numenta/htmresearch/blob/master/htmresearch/frameworks/pytorch/mnist_sparse_experiment.py#L65

Comment (subutai) [reply projects/speech_commands/experiments.cfg]: Good point. I'll fix these after Wed 😄

File: projects/speech_commands/run_experiment.py (hunk 1 above)
Comment (subutai) [reply projects/speech_commands/run_experiment.py]: Yes.
